docs/_core/borders/borders.stories.tsx

```diff
@@ -16,6 +19,4 @@ export default {
   },
 };
 
-export { Intro } from './intro.story';
-export { Examples } from './examples.story';
-export { Helpers } from './helpers.story';
+export { Intro, Examples, Helpers };
```

Comment:
Here's an example of the refactor, had to, otherwise `export { Intro } from './intro.story';` would be last